### DISCUSSION TITLE
A unique insight on the origin of the CMB

### DISCUSSION THREAD

**Discussion by LifeIsStrange** [a-cosmology-group/acg#275] [Astrophysics and Physics]
I have previously reviewed the evidence for the CMB being local, either being generated or altered in the solar system (kuiper belt, heliosphere, etc), in the local solar system neighbourhood (Local bubble, etc), or in the Milky Way here:
https://github.com/orgs/a-cosmology-group/discussions/237
The evidence, while indirect, is strong, in that the alignment of the CMB anisotropies (e.g. dipole) are aligned with the sun axis and with the milky way center, therefore suffering from very significant coincidence problems for a distant CMB explanation.

However I now believe that explanations of the CMB as having a local origin are 100% ruled out.
(a nearby interaction can still cause the anisotropies though (even though some anisotropies are corroborated at large distances)
As a reminder:
> the CMB frequency spectrum and its distortions should not be confused with the CMB anisotropy power spectrum, which relates to spatial fluctuations of the CMB temperature in different directions of the sky.[[1]](https://en.wikipedia.org/wiki/Cosmic_microwave_background_spectral_distortions#cite_note-1)

I have read about almost all past and upcoming CMB experiments, and there are rhoughly two kinds of them, and overwhelmingly, almost all incoming CMB experiments focus on the spatial anisotropies of the CMB, just like WMAP and PLANCK.
Those experiments (Simons, etc) will allow revolutionnary constraints on cosmological parameters, anisotropies, and will definitely rule out inflation models.
However the studies of the CMB temperature and its distortions has stagnated since decades and will mostly keep stagnating as future experiments are either not funded or will yield only moderate improvements (though essential to constrain CMB as generated by cosmic dust)
In BOTH cases, therefore ~all CMB experiments, are measured on earth or very close to the earth, therefore they mostly don't test the true distance of the CMB.
Very recently has been released the first CMB map by a distant instrument, JUNO on Jupiter:
https://arxiv.org/abs/2405.08388
This CMB map does not seem anomalous however If I understand correctly they have not measured the CMB temperature but only the spatial anisotropies..

Something very little known yet essential that I had ignored until now is that it is actually possible to measure the CMB temperature at a given redshift!!
It is in fact the first historical measurement of the CMB temperature, in 1941! decades before the discovery of the CMB, and this method, measuring the temperature of cyanogen in molecular clouds, is highly accurate, and was for a long time the best measurement of the CMB temperature.
Basically, there exist some molecules that are known to specifically absorb the CMB near its peak and therefore if measured with high precision, the temperature of the distant molecules allows to measure the temperature of the distant CMB.
While there are multiple molecules that have this property, Cyanogen is the most accurate CMB thermometer, sadly it has not yet been measured outside of the milky way.
The temperature measured is in excellent agreement with terrestrial CMB temperature measurements.
https://arxiv.org/abs/2409.18270
This is an almost irefutable proof that the CMB exists in the Milky way, beyond the solar system and that its temperature does not exhibit significant variance between distinct molecular clouds.
But this does not stops there, far from it:
Then the CMB has been measured in Carbon (or isotopes) at Z=1.77 at a temperature of 7.4 kelvins
https://www.nature.com/articles/371043a0.pdf
> Although the modern steady-state model 17 predicts TcMBR oc
(1 + z) until z ~ 6, other theories 18 make strong claims for a value
of TcMBR well below the standard prediction by z~3, which we
could check by such a measurement.

Measurement with H2
https://ui.adsabs.harvard.edu/abs/2020JPhCS1697a2013K/abstract
Measurement with CO (carbon monoxide) out to Z=3 !!
https://www.aanda.org/articles/aa/full_html/2011/02/aa16140-10/aa16140-10.html
https://ui.adsabs.harvard.edu/abs/2024ApJ...962..187E/abstract

And then came the ultimate CMB temperature measurement via H2O at Z=6.34 !!!
https://ui.adsabs.harvard.edu/abs/2022Natur.602...58R/abstract
With a CMB temperature between 16.4-30.2 K

Unless systematics falsify all those measurements (I very much doubt they can), this is direct empirical proof that the CMB has a distant origin until at least Z=6.34 (13 billions years ago)
Moreover, the CMB evolution with redshift is not random, it match the 1+Z relation just like cosmic time dilation.
Though the Beta paramater (the semantics of which I do not understand) is still not fully constrained empirically
cf https://hal.science/hal-04368837v2/document
and https://arxiv.org/abs/2002.12700 (also provide a constraint on cosmic opacity)
I did not know that but this precise and very significant CMB temperature evolution with redshift is predicted by LCDM. It is also predicted by a few alternative cosmologies (cf steady state mention).
The main alternative explanation of the CMB is that it is generated by cosmic dust, sadly papers about the topic does not seem to make predictions about the CMB as cosmic dust, temperature evolution over redshift. This could potentially falsify them.
Notably the cyanogen CMB temperature might be related to the hypothesis that they generate the CMB,
cf
> In 1993, K.C. Roth and other collaborators postulated a CMB
originated from interstellar molecular clouds due to excitation of cyanogen molecules [28]
https://adsabs.harvard.edu/full/1993ApJ...413L..67R
Even though he has not highlighted where they made that hypothesis

How this related to the missing dust signature problem and cosmic opacity should be urgently clarified
https://arxiv.org/abs/1706.04771
especially given this: https://arxiv.org/abs/2302.05468
and this https://arxiv.org/abs/2407.17599
The latter and https://indico.in2p3.fr/event/33359/attachments/85912/128933/marseille2024.pdf
shows that the CMB temperature might increase with matter density independently of redshift, if so, then assuming matter density has decreased with time (which fit the big bang but not only) then the CMB might still be regenerated by dust and evolve.

BTW an alternative probe is:
> CMB temperature can also be inferred from inverse Compton scatter-
ing of CMB photons off hot intracluster gas

https://ui.adsabs.harvard.edu/abs/2002ApJ...580L.101B/abstract
BTW a unique probe of the hubble tension would be to compare the anomalous coma cluster distance with the CMB temperature redshift of the Coma cluster!! This might be groundbreaking
https://arxiv.org/abs/2409.14546

previous post about cosmic birefringence
https://github.com/orgs/a-cosmology-group/discussions/231
https://indico.in2p3.fr/event/33359/
A wild question would be, can the CMB be generated by the overdensity of Little Red Dots?
https://ui.adsabs.harvard.edu/abs/2016PhRvD..93d3521A/abstract
https://ui.adsabs.harvard.edu/abs/2022A%26A...666L...3G/abstract
https://ui.adsabs.harvard.edu/abs/2024PhRvD.109b3534A/abstract
https://ui.adsabs.harvard.edu/abs/2024arXiv240506374D/abstract
> https://www.sciencedirect.com/science/article/pii/S2212686424000761?via%3Dihub
upcoming high redshift CMB temperature measurements will allow record constraints on alternative cosmologies, including steady state and varying speed of light
btw: The first dedicated
redshift drift experiment is presently ongoing, using the ESPRESSO
spectrograph, and its first results (expected in the coming months) will
provide an important assessment of the feasibility of future detections. ????

BTW an alternative to measuring the redshift drift would be to measure the CMB temperature lowering over years as a proxy measure of the redshift drift!
An intuitive calculation shows that this would become possible to measure with only a 100 times improvement in our CMB temperature measurement at local redshift, and might be even easier at higher redshift to constrain dark energy, the former, this new redshift drift metric, is just about to happen:
https://arxiv.org/pdf/2307.08513 It is kind of mind blowing nobody thought of this realtime alternative to redshift drift before!
Also quid about luminosity distance and angular distance drift?

note that apparently it is possible to infer the CMB temperature at a distance via the SZ effect though I have always been dubious of that (how do we prove it is not the local inprint of clusters light that affect the CMB versus that the CMB truly passed through those clusters? Maybe via thompson scattering inferences?), moreover SZ does not yet allow to reach high redshift.
https://arxiv.org/abs/0704.0221
https://ui.adsabs.harvard.edu/abs/2024arXiv240912188S/abstract
http://viavca.in2p3.fr/presentations/cmb_and_ether_drift_experiments.pdf
https://academic.oup.com/mnras/article/443/3/1881/1079142
https://arxiv.org/abs/2308.07529

**Comment by HanDeBruijn** [comment]
Can you please **summarize** the above for someone who's English is good enough for concatenating mathematical formulas and that's it?

**Reply by LifeIsStrange** [reply]
The CMB temperature can be measured at high redshift because some molecules specifically absorb the CMB peak and hence assuming this is their main heat source, their temperature become an accurate proxy of the CMB temperature.
The CMB is therefore empirically proven to have existed 13 billion years ago and its temperature to be multiplied by the redshift (1+z).
This rule out any nearby or recent origin of the CMB. The (1+z) scaling relation rule out many alternative cosmologies.
I have given additional info but this is the main gist

**Reply by HanDeBruijn** [reply]
>The (1+z) scaling relation rule out many alternative cosmologies.

Yes, and my own [CMB model](http://www.alternatievewiskunde.nl/cosmology/CMBR.htm) is among them. Thanks for the summary anyway.

**Comment by sahil5d** [comment]
I don't think the CMB temp increases with distance. ACG has established that receding-galaxies-expanding-space is false, so in a model where space is space (non-expanding), it's unlikely for a very distant emitter's radiation to get cooler and cooler and when it reach the Milky Way just happen to have the properties of a perfect blackbody spectrum (Planck spectrum), shape and photon count. A redshifted Planck spectrum is not a Planck spectrum.

Also this topic was discussed on these threads
https://github.com/orgs/a-cosmology-group/discussions/178#discussioncomment-7435822
https://github.com/orgs/a-cosmology-group/discussions/262#discussioncomment-10573685

**Reply by ExpEarth** [reply]
@LifeIsStrange Thanks for all this compilation of studies.  The CMB temperature measurements using CO, etc. have been discussed here a few times.  I have an explanation for them in my shell universe model #256 .  In my model the Milky Way is close to the centre of the shell but not right on it.  So if the CMB temperature keeps dropping past our position, it could in theory be measured.

Question:  In the Planck study they found evidence of a 'cool spot' in the CMB in the southern hemisphere.  That spot can be interpreted as an underdensity in matter, which would be consistent with my shell model (where there is a relative void at the origin).  Do you (or anyone) know if the remote CMB temperature measurements are from the northern or southern hemispheres?  If the studies were in the northern, it would be consistent with the cool spot.    

@sahil5d Yes, it's true we have discussed this many times, but that does not mean there was any resolution on it.  I am with @LifeIsStrange that these CMB temperature measurements prove that the CMB does not arise locally, but with a caveat.  I have already given one explanation for how the CMB can cool and remain blackbody radiation as it moves from high redshift towards us, in my shell universe model.  The mechanism depends on equilibration of CMB wave energy with a photonic spacetime.  By that model you can actually say that the CMB is arising locally, since the CMB photon number changes at each point.

**Reply by LifeIsStrange** [reply]
The cold spot is not necessarilly caused by an underdensity of matter, it could also be caused by an overdensity, especially of microwave opacifying molecules. Also the cold spot is on the southern sky. The cmb temperature measurements have been made on multiple observations, it seems likely at least one of them was  in the northern sky. Moreover, no they are most likely not aligned with the cold spot and since the temperature increase with redshift I don't see the logic of the effect of the cold spot.
The supervoid explanation of the cold spot has almost been ruled out as the number density of observed galaxies in the region is not low.
> Although large voids are known in the universe, a void would have to be exceptionally vast to explain the cold spot, perhaps 1,000 times larger in volume than expected typical voids. It would be 6 billion–10 billion [light-years](https://en.wikipedia.org/wiki/Light-year) away and nearly one billion light-years across

There is a 2015 study and a DESI hint but except that, most studies shows no underdensity or void alignment.

IMO the best study on the cold spot is this
https://indico.in2p3.fr/event/33359/attachments/85912/128933/marseille2024.pdf
https://indico.in2p3.fr/event/33359/
They detect a new CMB foreground (correlated to a population of HI deficient galaxies) and when they correct for it, the cold spot disappear (page 32)
the nature of the foreground they identified is unclear (molecular absorption or new physics)
> Surroundings of late-type Spiral galaxies have sistematically
lower CMB temperatures

previous paper by same author
https://arxiv.org/abs/2310.13755
the anomalous cmb temperature decrement has also been observed more recently
https://arxiv.org/abs/2407.17599

**Reply by HanDeBruijn** [reply]
@sahil5d .
>A redshifted Planck spectrum is not a Planck spectrum.

That depends. See [this reply](https://github.com/orgs/a-cosmology-group/discussions/262#discussioncomment-10578831) in thread 262.

**Reply by ExpEarth** [reply]
@LifeIsStrange

Thanks for the detailed reply.  The general relationship in CMB temperature that you describe is a feature of my model, where the outer 'shell' region is at about 29 K and the central point could be near 0.  We are not right at the centre point, and so our local CMB is at 2.7 K (figure below).  So it might be possible to check for the validity of my model my making measurements of CMB temperature in the region between us and the centre point.
![image](https://github.com/user-attachments/assets/0c783db4-7473-4905-aeb1-fc9f1373d0d0)

Getting back to the cold spot, is it is in the southern sky, then perhaps it is in a region closer to the centre than we are.  Perhaps most measurements of CMB temperature have been made in the norther sky, and so these will be hotter.  The cool spot could align with lower CMB temperatures close to the centre.  I do see your papers positing other causes for the cold spot.  I did also notice a 2016 paper (section 2.5) where they say it could be caused by an underdensity, but that it could require a new cosmological model (which I have).  That would also be consistent with my model, which calls for a relative void near the centre.
https://iopscience.iop.org/article/10.1088/0264-9381/33/18/184001

**Reply by sahil5d** [reply]
@HanDeBruijn your linked message is about the standard model (Big Bang LCDM) explanation. ACG has established that receding-galaxies-expanding-space is false.

@ExpEarth Your explanation was
> Basically, it requires a specific model of spacetime, in which spacetime is photonic. This is all part of my CMB gravity model too. The the energy of inwardly moving photons is absorbed by spacetime, due to the gravitational (Hubble) redshift. The CMB waves decrease in photon number and photon energy, as energy is redistributed with spacetime photons.

In my opinion, that isn't explanatory. Your model has swept a hypothetical 29K photon count under the rug of "absorbed by spacetime". I would be willing to entertain that energy per photon is absorbed (and something nearby heats up in an energy conserving way), but you also claim that the number of photons decreases in just the way that a 29K Planck spectrum becomes a 2.7K Planck spectrum, and that's a colossal special mechanism.

**Reply by HanDeBruijn** [reply]
@sahil5d .
>your linked message is about the standard model (Big Bang LCDM) explanation. ACG has established that receding-galaxies-expanding-space is false.

Nope. It can also be about e.g. a Variable Mass explanation in a Static Euclidean Universe, as long as

$$
\frac{\lambda_0}{\lambda} = \frac{\nu}{\nu_0} = \frac{\Theta}{\Theta_0} \left( = \frac{m}{m_0} \right)
$$

where $\lambda=$ wavelength, $\nu=$ frequency, $\Theta=$ temperature, $m=$ mass, $_0=$ here and now.

**Reply by sahil5d** [reply]
There are a few more issues
- The shell model says we're in black hole, but it's not actually a black hole. There's stuff inside.
- The model claims that near the shell must be younger galaxies. But JWST images show that far away are Milky-Way-like galaxies.
- Big Bang CMB-temp-with-distance studies are flawed.
  - "Assuming the CMB is the only source of excitation, Songaila et al. (1994) measured T_CMB", "assumed the CMB to be the only source of excitation to provide stringent upper-limits on T_CMB" https://www.aanda.org/articles/aa/full_html/2011/02/aa16140-10/aa16140-10.html#:~:text=Assuming%20the%20CMB%20is%20the%20only%20source%20of%20excitation
  - "Other parameters of the fit were fixed to T_CMB = 11.6 K" https://iopscience.iop.org/article/10.1088/1742-6596/1697/1/012013/pdf
- The model has a supermassive radiating shell, which should dissipate but doesn't because of a caveat that radiation "does not substantially leak out to an exterior space", but at the same time "I'm just allowing for some escape of radiation, while keeping the notion of a black hole". But that's not what a black hole does. Black holes (or rather, the ultra massive objects at the centers of galaxies) emit jets of matter and gamma rays and they glow in the radio spectrum.

**Reply by sahil5d** [reply]
@HanDeBruijn the problems with VM are on that thread. Your last reply was this 
https://github.com/orgs/a-cosmology-group/discussions/262#discussioncomment-10630237.?

Here are some problems
- The galaxies very far away do not all have "VM younger matter". JWST observes a diversity of galaxies far away (both young, developing and old, highly developed).
- You should have some proof that the "VM Sun" 1.4 x'ed its mass (e^(4.6 / 13.8)) and somehow planetary orbits continued for 4.6 Gy and somehow the T^2 ~ R^3 relation stands today.
- Why do you say the VM hypothesis means that the Earth-Moon distance increases? Your hypothesis implies the Earth's mass increases, so that should increase the force of gravity and reduce the orbit radius.
- There is a problem with VM's "radiation is redshifted at the source and not along the way". As you claim that mass everywhere is continuously increasing, then so should the hf/c^2 of a photon. So a photon in flight should be increasing its hf/c^2, and thus its frequency! So a photon even though "VM redshifted" at the source, would be "VM hf/c^2 boosted" during travel, and thus arrive with no detectable redshift, contradicting observation.

**Reply by HanDeBruijn** [reply]
@sahil5d .
My last reply in that thread concerned our mutual misunderstanding, which seems to be insurmountable.
The VM theory is much more involved than can be explained to Homo Zappiens in half a minute.
Main problem is that it also.affects our measuring devices, which leads to [uncommon result](https://github.com/orgs/a-cosmology-group/discussions/265#discussioncomment-10749472)s.
There is no other way than study the whole thing. I'm sorry.

>As you claim that mass everywhere is continuously increasing, then so should the hf/c^2 of a photon

Short answer: photons are not corpuscles.

**Reply by sahil5d** [reply]
VM's hypothetical increasing mass from nowhere is a colossal special mechanism.

At minimum you should be able to explain problem number 1, the distant galaxies. You know those galaxies showed spectral lines of oxygen. Such galaxies, at time of emission, were not young.

**Reply by ExpEarth** [reply]
@sahil5d
 `The shell model says we're in black hole, but it's not actually a black hole. There's stuff inside.`
Yes, I use the term black hole, both for the shell universe and for other kinds of 'black holes', but I emphasize that they are not the black holes with singularity commonly envisioned.  There _is_ stuff inside them.  I argue that the processes of the shell universe can be used to understand 'ordinary' black holes.

`The model claims that near the shell must be younger galaxies. But JWST images show that far away are Milky-Way-like galaxies.`
The shell universe grows only slowly through accretion at the edge.  So the very farthest galaxies are likely to be younger.  But the galaxies within the main part of the shell can be far older than 13.8 billion years.

`Big Bang CMB-temp-with-distance studies are flawed. `
We have been over this a few times.  Certain assumptions of the LCDM model are used in the analysis, but those assumptions do not necessarily have the effect of disproving the conclusions.  No one in ACG has taken the time to actually show how these assumptions would negate the conclusions.  It could be a difficult task though.

`But that's not what a black hole does. Black holes (or rather, the ultra massive objects at the centers of galaxies) emit jets of matter and gamma rays and they glow in the radio spectrum.`
This is what the supermassive black holes/AGNs do.  This is because they have high rotation.  The rotation state of the universe is not known, but if anything it must be slower than for AGNs.

**Reply by sahil5d** [reply]
> they are not the black holes with singularity commonly envisioned... I argue that the processes of the shell universe can be used to understand 'ordinary' black holes.

Ordinary black holes (or the dense things at the center of galaxies) don't have matter within their event horizon. So I think there is no way your shell model is connected to or can help understand ordinary black holes.

> So the very farthest galaxies are likely to be younger.

Will you make a prediction and stick to it? We already observe developed galaxies at z=14.

> We have been over this a few times. Certain assumptions of the LCDM model are used in the analysis, but those assumptions do not necessarily have the effect of disproving the conclusions. No one in ACG has taken the time to actually show how these assumptions would negate the conclusions.

We have, and you have ignored both Marmet's and my responses. The big bang CMB studies assume the whole conclusion, and therefore their conclusion has no merit. In the second paper I linked (iop.org), the graphed "measurements" are not measurements but _estimates_, specifically calculations where one of the inputs is a measurement and one of the inputs is a theoretical value. "Other parameters of the fit were fixed to T_CMB = 11.6 K". Where did that "11.6 K" come from??

> This is what the supermassive black holes/AGNs do. This is because they have high rotation. The rotation state of the universe is not known, but if anything it must be slower

That still leaves the radio glow, and the hypothetical shell would dissipate.

**Reply by ExpEarth** [reply]
@sahil5d 
`Ordinary black holes (or the dense things at the center of galaxies) don't have matter within their event horizon. So I think there is no way your shell model is connected to or can help understand ordinary black holes.`
I don't follow you here.  Are you referring to the donut-shaped images of the SMBHs that have appeared?  The dark space arises as the rotation forces them into a torus.

`Will you make a prediction and stick to it? We already observe developed galaxies at z=14.`
Observations are coming in faster than I can predict them.  Don't forget the little red dots at the highest redshifts.

`not measurements but estimates, specifically calculations where one of the inputs is a measurement and one of the inputs is a theoretical value. "Other parameters of the fit were fixed to T_CMB = 11.6 K". Where did that "11.6 K" come from??`
I only keep ignoring you because you do not show how that one assumption affects the results.  like I said, it's not an easy exercise to show it.  And the redshift-temperature relationship has various ways of measuring, not just that one.

`That still leaves the radio glow, and the hypothetical shell would dissipate.`
There could be some losses to the outside, but there are gains too from mass and radiation outside.

**Reply by sahil5d** [reply]
> Are you referring to

I'm referring to the idea that these ultra massive objects have a radius like an event horizon within which there is no matter.

> Observations are coming in faster than I can predict them. Don't forget the little red dots at the highest redshifts.

This is very telling.

We're not going to believe your hypothesis by waiting for the observation and then waiting for you to make a claim. Credibility does not come from postdiction.

> I only keep ignoring you because you do not show how that one assumption affects the results.

I have shown. For example, in the paper linked, the "11.6 K" came not from observation. Red flag!

> There could be some losses to the outside, but there are gains too from mass and radiation outside.

Another special mechanism...

**Reply by HanDeBruijn** [reply]
@sahil5d .
>VM's hypothetical increasing mass from nowhere is a colossal special mechanism.

It's Arp and Narlikar's invention, not mine. However, the "colossal" depends on how you look at it. In Orbital time $T$ there is "sudden" creation at $T=-A$. In Atomic time $t$ there is **slow** exponential growth of (elementary particle rest) mass. The two timeframes are related by

$$
1+T/A = e^{t/A} \quad \mbox{with} \quad A = \mbox{"age"}
$$

It is noticed that a colossal [Time Dilated Past](https://github.com/orgs/a-cosmology-group/discussions/160) is involved indeed at large distances.

>At minimum you should be able to explain problem number 1 [ .. ] those galaxies showed spectral lines of oxygen. 

The spectral lines of that oxygen are very redshifted. According to the VMT this means young electrons and young protons in the atom.

>Ordinary black holes (or the dense things at the center of galaxies) don't have matter within their event horizon. 

Then you know more about the inside of those _ordinary black holes_ than I do, Sahil.
Apart from this, I rather think that ACG should restrict to the original Laplacian definition: _a black hole is a body so heavy that no light can escape from it_ and that's all.

>Will you make a prediction and stick to it? We already observe developed galaxies at z=14.

The VMT makes a prediction of $z_{max} = 28.5$ . And @HanDeBruijn sticks to it.

**Reply by sahil5d** [reply]
> The spectral lines of that oxygen are very redshifted. According to the VMT this means young electrons and young protons in the atom.

But those oxygen-forming electrons and protons are not young. Oxygen-forming stars are not early in a galaxy's life. The time when electrons and protons are "young" is before a galaxy's stars ignite.
